run-perf

```diff
@@ -36,7 +37,6 @@ export MAIN_JOBEVENT_UNIQUE_SIZE=20000
 export OUTPUT_DATE_FROM=2026-02-01
 export OUTPUT_DATE_TO=2026-03-01
 
-/usr/bin/time --format='\n\nmemory %MK\ncpu %P\ntime %es\n\n' \
```

Comment:
removed too much?

This should still be here, just, using `$TIME`

Reply:
Thanks, corrected :) and do you think I should change anything in the dev setup at this point? or does the updated script speak for itself

Reply:
Looking at the diff in https://github.com/ansible/metrics-utility/pull/125/files#diff-131add6a5e2097d535cac484706b46ea3a5d32c9697485fa1161d4c4173086b1 ..

I still see a couple of removed calls

But otherwise good I think :)